### PR TITLE
:art: Fix padding around app name in startup message when containing non-ascii characters

### PR DESCRIPTION
**Pull request by Maxi-Mega**
When the app name contains non-ascii chars (that are stored on more than 1 byte) , the padding around it in the startup message becomes incorrect. The veritable length of the app name can be obtained by getting the length of the underlying rune slice, like `len([]rune(s))`.

I added a test (`Test_App_Master_Process_Show_Startup_MessageWithAppNameNonAscii`) that creates an app with a name containing accentuated letters.

### CODE REVIEW

**Comment by ReneWerner87** [comment]
@Maxi-Mega can you refresh ? after the merge of https://github.com/gofiber/fiber/pull/1977 there are conflics

**Comment by Maxi-Mega** [comment]
> @Maxi-Mega can you refresh ? after the merge of #1977 there are conflicts

Done, tell me if everything's ok

File: app_test.go [hunk 1]
```diff
@@ -1401,6 +1401,16 @@ func Test_App_Master_Process_Show_Startup_MessageWithAppName(t *testing.T) {
 	utils.AssertEqual(t, true, strings.Contains(startupMessage, app.Config().AppName))
 }
 
+func Test_App_Master_Process_Show_Startup_MessageWithAppNameNonAscii(t *testing.T) {
```
Comment (efectn) [inline app_test.go:1404]: Can you move all startup message, route list tests to listen_test.go

**Review by efectn — CHANGES_REQUESTED** [review]

File: app_test.go (hunk 1 above)
Comment (Maxi-Mega) [reply app_test.go:1404]: > Can you move all startup message, route list tests to listen_test.go

Sure

**Review by efectn — APPROVED** [review]

**Pull request merged**